src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -457,4 +457,19 @@
   <data name="MissingKeepAlive" xml:space="preserve">
     <value>Missing argument for '/keepalive' option.</value>
   </data>
+  <data name="AnalyzerTotalExecutionTime" xml:space="preserve">
+    <value>Total analyzer execution time: {0} seconds.</value>
+  </data>
+  <data name="MultithreadedAnalyzerExecutionNote" xml:space="preserve">
+    <value>NOTE: The actual wall clock time is likely lesser due to multithreaded analyzer execution.</value>
```

Comment:
Remove "actual" (it doesn't add anything).
"less" not "lesser". Or "lower".

How about:
> NOTE: The elapsed time may be lower because analyzers may run concurrently.

?